pkg/sql/logictest/testdata/logic_test/alter_primary_key

```diff
@@ -7,8 +7,10 @@ CREATE TABLE t (x INT PRIMARY KEY, y INT NOT NULL, z INT NOT NULL, w INT, INDEX
 statement ok
 INSERT INTO t VALUES (1, 2, 3, 4), (5, 6, 7, 8)
 
-statement ok
+query T noticetrace
 ALTER TABLE t ALTER PRIMARY KEY USING COLUMNS (y, z)
+----
```

Comment:
Can you add some dummy tests that print multiple notices (like using the notice builtin) and see how this interacts?

Reply:
unfortunately `crdb_internal.notice` doesn't work here because it executes on remote nodes, which notices currently do not support :(

Reply:
i can make this "double test" in the follow up PR

Reply:
sorry I'm not following -- can't you make that test in a `local` only config so it executes on the same node?

Reply:
didn't work on local either and i found out why...because lib/pq is the worst.